modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+
+[id="installation-osp-deploying-bare-metal-machines_{context}"]
+= Deploying a cluster with bare metal machines
+
+If you want your cluster to use bare-metal machines, modify the

```

Comment:
Is it bare metal or bare-metal? We mostly use the former.

Reply:
Bare metal in this context. "Bare-metal" only if it's used in a phrase like "bare-metal compute machines."